tests/spec/test_dagspec.py

```diff
@@ -320,6 +321,47 @@ def test_doesnt_load_env_in_default_location_if_loading_from_dict(tmp_nbs):
     assert set(spec.env) == {'user', 'cwd', 'root', 'now'}
 
 
+def test_error_if_env_yml_exists_cwd(tmp_directory):
+    Path('env.yml').write_text("{'a': 1}")
+    Path('script.py').write_text("""
+# + tags=["parameters"]
+something = None
+""")
+
+    Path('pipeline.yaml').write_text("""
+tasks:
+    - source: script.py
+      product: output.ipynb
+""")
+
+    with pytest.raises(DAGSpecInitializationError) as excinfo:
+        DAGSpec('pipeline.yaml')
+
+    assert 'Error: found env file ends with .yml' in str(excinfo.value)
+
+
+def test_error_if_load_env_yml_from_sibl_dir(tmp_directory):
```

Comment:
Add same tests for pipeline.yml (try to parameterize those functions instead of duplicating).